pages/participantinfo.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+    Button,
+    Box,
+    Center,
+    Text,
+    Input,
+    FormControl,
+    FormLabel,
+    Progress,
+    Stack,
+    HStack,
+    VStack,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+/**
+ * This is the page that a user will use to enter the participants personal information
+ * onto the SDC platform as a parent of volunteer
+ */
+export default function Participantinfo(): JSX.Element {
+    const [progressBar, setprogressBar] = useState(Number);
+    if (progressBar <= 0) {
+        setprogressBar(14);
+    }
+    return (
+        <Center h="500px">
```

Comment:
Is there a reason why you want to set a fixed height of `500px` for the container?